src/interned.rs

```diff
@@ -73,7 +73,7 @@ pub struct Value<C>
 where
     C: Configuration,
 {
-    data: C::Data<'static>,
+    pub data: C::Data<'static>,
```

Comment:
it feels like this should be renamed to `fields` for consistency; maybe `C::Fields`, even